src/main/java/net/minecraftforge/common/ForgeModContainer.java

```diff
@@ -404,6 +404,11 @@ public void missingMapping(RegistryEvent.MissingMappings<Item> event)
                 ResourceLocation newTotem = new ResourceLocation("minecraft:totem_of_undying");
                 entry.remap(ForgeRegistries.ITEMS.getValue(newTotem));
             }
+            if (entry.key.toString().equals("forge:bucketFilled"))
```

Comment:
I'm pretty sure it ends up being lower cased because the registries do not support capital letters.

Reply:
Can't we just use equalsIgnoreCase